SDK/AppCenter/Microsoft.AppCenter.Shared/AppCenter.cs

```diff
@@ -11,50 +13,71 @@ namespace Microsoft.AppCenter
     /// </summary>
     public partial class AppCenter
     {
-        // Gets the first instance of an app secret corresponding to the given platform name, or returns the string 
+        const char SecretDelimiter = ';';
+        const char PlatformKeyValueDelimiter = '=';
+        const string TargetPostfix = "Target";
+        const string SecretPostfix = "appSecret";
+
+        // Gets the first instance of an app sceret and/or target token corresponding to the given platform name, or returns the string 
         // as-is if no identifier can be found. Logs a message if no identifiers can be found.
-        internal static string GetSecretForPlatform(string secrets, string platformIdentifier)
+        internal static string GetSecretAndTargetForPlatform(string secrets, string platformIdentifier)
         {
+            var platformTargetIdentifier = platformIdentifier + TargetPostfix;
             if (string.IsNullOrEmpty(secrets))
             {
                 throw new AppCenterException("App secrets string is null or empty");
             }
 
             // If there are no equals signs, then there are no named identifiers, but log a message in case the developer made 
             // a typing error.
-            if (!secrets.Contains("="))
+            if (!secrets.Contains(PlatformKeyValueDelimiter.ToString()))
```

Comment:
`PlatformKeyValueDelimiter` can be declared as string instead of char because C# string split accept string as well. Then you don't have to `.ToString()`.

Reply:
Same goes to char `SecretDelimiter`.